src/ray/raylet/task_dependency_manager.h

```diff
@@ -143,7 +153,7 @@ class TaskDependencyManager {
   /// transfer or reconstruction. These are objects for which: (1) there is a
   /// subscribed task dependent on it, (2) the object is not local, and (3) the
   /// task that creates the object is not pending execution locally.
```

Comment:
Could you document the arguments here? Especially if `transfer_required` is an output.